tools/autograph-client/client.go

```diff
@@ -130,6 +130,7 @@ examples:
 	flag.StringVar(&cn, "cn", "", "when signing XPI, sets the CN to the add-on ID")
 	flag.Var(&algs, "c", "a COSE Signature algorithm to sign an XPI with can be used multiple times")
 	flag.StringVar(&pk7digest, "pk7digest", "sha1", "an optional PK7 digest algorithm to use for XPI file signing. Defaults to 'sha1'")
+	flag.StringVar(&rootPath, "r", "/path/to/root", "Path to a PEM file of root certificates")
```

Comment:
nit: I don't remember if the default value shows up in the help or usage output, but if it does, should it be `/path/to/root.pem`?

Reply:
Good call! I think it does, good to clarify that root is file and not a dir in code regardless :+1: